modules/ROOT/pages/shared-bucket-access.adoc

```diff
@@ -95,6 +95,20 @@ Document operations also have a different impact on tombstones when Shared Bucke
 |Removes the tombstone metadata
 |===
 
+== FAQ
+
+=== How do I query for all documents in a channel with N1QL?
+
+// TODO
```

Comment:
@adamcfraser do we have an example N1QL statement to answer this question?

Reply:
Please check : https://forums.couchbase.com/t/n1ql-sync-gateway/18427/2

I would also add the statement WARNING user that the sync metadata structure could change at any point and applications should not rely on that to drive their business logic. This is a convenience for introspecting their document sync metadata.

Reply:
Ok thanks!